docs/getting_started.rst

```diff
@@ -33,8 +33,8 @@ Installation Quick Reference Guide
 
          python nengo_fpga/id_extractor.py <board>
 
-   where **<board>** is the board name as it appears in ``fpga_config``.
-   See the `Copy Protection`_ section for more information.
+    where **<board>** is the board name as it appears in ``fpga_config``.
```

Comment:
This rendered weird on my build. The one just below it rendered fine though... so maybe a tabbing issue?

Reply:
![image](https://user-images.githubusercontent.com/1971583/62153748-43644880-b2d3-11e9-92a4-59059429ee71.png)